share/config.py

```diff
@@ -290,6 +290,11 @@ def add_output(self, output_type: str, **kwargs: Any) -> None:
 
         output: Optional[Output] = None
         if output_type == "elasticsearch":
+            if "es_index_or_datastream_name" in kwargs:
```

Comment:
Can we link to an issue where this breaking change is detailed and the action item of removing this check at a later version?

Reply:
https://github.com/elastic/elastic-serverless-forwarder/issues/116